<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWidget</class>
 <widget class="QWidget" name="SimulationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget_Hpm,#widget_Environment,#widget_Model{
	background-color: rgb(255, 255, 255);
border : 2px solid black;
}</string>
  </property>
  <widget class="QWidget" name="widget_Environment" native="true">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>171</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="3" column="1">
     <widget class="QLabel" name="label_Value8">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>辐射源位置：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_Value7">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>160</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>天气：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>雨衰系数：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_Value9">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>无人机位置：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_Value10">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(62, 155, 255);
color: rgb(255, 255, 255);
font: 75 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>攻击环境设置</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_Hpm" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>211</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="horizontalSpacing">
     <number>5</number>
    </property>
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <item row="6" column="1">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>波束宽度：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3" colspan="2">
     <widget class="QLabel" name="label_Value2">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>发射频率：</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>入射方向：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QLabel" name="label_Value3">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3" colspan="2">
     <widget class="QLabel" name="label_Value5">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(62, 155, 255);
color: rgb(255, 255, 255);
font: 75 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>高功率辐射源</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="3" colspan="2">
     <widget class="QLabel" name="label_Value4">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3" colspan="2">
     <widget class="QLabel" name="label_Value6">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>波形：</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>极化方向：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLabel" name="label_Value1">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QWidget" name="widget_plot" native="true"/>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>辐射功率：</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_Model" native="true">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>60</y>
     <width>131</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(62, 155, 255);
color: rgb(255, 255, 255);
font: 75 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>模型选择</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>无人机：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_Value11">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>天线：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_Value12">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>线缆：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_Value13">
      <property name="text">
       <string>未设置</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>192</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_Setting" native="true">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>70</y>
     <width>166</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5"/>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>168</width>
     <height>345</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>166</width>
      <height>343</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_35">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(62, 155, 255);
color: rgb(255, 255, 255);
font: 75 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>仿真设置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>仿真方法：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>TDIE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MoM</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>250</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
